integration-tests/mysql_test.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("MySQL", Label("MySQL"), func() {
 			Expect(mockTerraform.FirstTerraformInvocationVars()).To(
 				SatisfyAll(
 					HaveKeyWithValue("db_name", "csb-db"),
-					HaveKeyWithValue("authorized_network_id", ""),
+					HaveKeyWithValue("authorized_network_id", BeAssignableToTypeOf("")),
```

Comment:
thoughts: this expression: `HaveKeyWithValue("authorized_network_id", BeAssignableToTypeOf("non empty string"))` would be true and it seems we are not testing what we want